### PR TITLE
Zookeeper Upgrade Testing Scenario

### PR DESCRIPTION
**Pull request by JumaX**
# Description

This PR adds a new upgrade testing script, to specifically test zookeeper upgrades when TLS is enabled.  This has been introduced due to ANSIENG-967.

Fixes # (issue)

ANSIENG-967

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

The script has been run successfully and validates the that fix implemented in the following PR works:

https://github.com/confluentinc/cp-ansible/pull/782/files

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] Any variable changes have been validated to be backwards compatible

### CODE REVIEW

File: roles/confluent.test/upgrade_testing/zookeeper-tls-upgrade-620-current.sh [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+cd ..
+
+## Checkout starting branch
+echo "Checking out $START_BRANCH branch"
+git checkout $START_BRANCH
+
+## Run Molecule Converge on scenario
+echo "Running molecule converge on $SCENARIO_NAME with tags $TAG_NAME"
+molecule converge -s $SCENARIO_NAME $TAG_NAME
+
```
Comment (nsharma-git) [inline roles/confluent.test/upgrade_testing/zookeeper-tls-upgrade-620-current.sh:32]: Should we not run the verify.yml at pre and post upgrade state?

**Review by nsharma-git — COMMENTED** [review]
Any specific reason to have only zookeeper in the upgrade testing. Should we not have all components to ensure there are no further impact of the upgrade.

**Comment by JumaX** [comment]
We have other upgrade tests to test all components, this was specifically to test zookeeper with TLS, which we previously had no testing for during an upgrade.  We could include additional components but not really much point as we test this elsewhere and it slows down this test.

File: roles/confluent.test/upgrade_testing/zookeeper-tls-upgrade-620-current.sh (hunk 1 above)
Comment (JumaX) [reply roles/confluent.test/upgrade_testing/zookeeper-tls-upgrade-620-current.sh:32]: We do not run verification after upgrades as we do not modify configs in any way, we simply upgrade the binaries.  If the upgrades fail to work the health checks will fail on the given component.

**Review by nsharma-git — APPROVED** [review]

**Pull request merged**